default.nix

```diff
@@ -0,0 +1,24 @@
+{ pkgs ? import <nixpkgs> (import (builtins.fetchTarball https://github.com/input-output-hk/haskell.nix/archive/master.tar.gz))
```

Comment:
Ideally you'll want to pin nixpkgs as well. Otherwise you'll need to keep rebuilding GHC, etc everytime channel advanced.